src/ragas/validation.py

```diff
@@ -67,6 +67,7 @@ def validate_column_dtypes(ds: Dataset):
     EvaluationMode.ga: ["ground_truth", "answer"],
     EvaluationMode.qga: ["question", "ground_truth", "answer"],
     EvaluationMode.qcg: ["question", "contexts", "ground_truth"],
+    EvaluationMode.ts: ["text", "summary"],
```

Comment:
I have one question here. would it make sense to make it ["contexts"] and summary?

the reasoning was that 'text' and 'contexts' object seems similar and I was wondering if we could join them together

what do you think @sky-2002 @shahules786 ?

Reply:
Yes.

Reply:
@jjmachan @shahules786 Done